### ISSUE TITLE
Make an instance readonly / locked

### ISSUE THREAD

**Issue by kof** [webstudio-is/webstudio#3494]
As a user I want to prevent any further modifications of an instance so it can not be accidentally modified: styles, properties, children.

It could be a lock button like in figma

**Comment by figotech** [comment]
also would be amazing to be able to lock tokens

**Comment by kof** [comment]
@figotech this idea has surfaced a few times,curious how would you use it, would you just introduce a second click for unlocking or would you make it permissions-based so someone can't actually change it.

Is it just to avoid accidental change or to actually prevent certain user types from modifying tokens?

**Comment by figotech** [comment]
i think just second click is enough. Pretty much the reason its on my mind is:
We have some container, wrapper, or just general style tokens, lets say the usual container with margin 0 auto;  A LOT of times we go back to a certain section that may need paddings or other adjustments and sometimes in the rush dont check that the container token is selected isntead of local or another token, needless to say this messes up every other container.  Sometimes the token modified is not as obvious in the particular page you are changing. 
The simple lock is mostly to prevent mistakes not to lock out the user at all.

I think also a visual cue on the properties modified by a locked token (a slight variation of the orange used when another entity is modifying a variable ) could be helpful if possible

I think also slots and honestly almos t every instance could benefit from this, you could be modifying lets say a slide, then you got some essential parts ready, lock those up, then mess around with more properties, you could have besides the "clear all styles" option a "clear all unlocked styles" one and essentialy use locking to "snapshot" certain progress.

Again no need for a complciated permisisons system just a basic measure against potential mistakes and helping new developers not mess up while they learn about how tokens work.

**Comment by kof** [comment]
@figotech makes sense. So the use case is self-protection against accidental change.
